packages/application-shell/src/index.js

```diff
@@ -3,7 +3,8 @@
  */
 import './public-path';
 
-export { default } from './components/application-shell';
```

Comment:
Can we keep it the `default`? I consider this to be the **main** component of the package and the rest are additional/complementary stuff that gets exported, so there is like a distinction of which component is more important to use.

But it's not super important if there are good reasons to refactor it...

Reply:
I guess we could keep both for now. But in general, it's not a great idea to expose both a default and named exports from a single module

https://github.com/rollup/rollup/issues/1961

Reply:
Ah I see, it's mostly for CJS support. Alright, it's fine by me then 💯

Reply:
I would not reccomend that either. I also do not mind a full and painful refactor across all apps with this maybe even not being backwards compat. It should be a do-able effort and a long standing refactor passed onto teams might be in sum a bigger effort than a at once version.